src/Controller/Site.php

```diff
@@ -1,66 +1,36 @@
 <?php
 
-namespace Bolt\Docs;
+declare(strict_types=1);
 
-use Silex;
-use Silex\Api\ControllerProviderInterface;
+namespace Bolt\Docs\Controller;
+
+use Bolt\Docs\Documentation;
+use Bolt\Docs\Page;
+use Bolt\Docs\Version;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
-class Controllers implements ControllerProviderInterface
+/**
+ * Site controller.
+ *
+ * @author Carson Full <carsonfull@gmail.com>
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class Site extends AbstractController
```

Comment:
Shouldn't `Documentation` be passed into the class instead since it is needed for every method?

Reply:
WFM … done